vizro-core/pyproject.toml

```diff
@@ -82,4 +81,4 @@ filterwarnings = [
   "ignore:When grouping with a length-1 list-like, you will need to pass a length-1 tuple to get_group:FutureWarning"
 ]
 norecursedirs = ["tests/tests_utils", "tests/js"]
-pythonpath = ["tests/tests_utils"]
+pythonpath = ["tests/tests_utils", "examples/kpi"]
```

Comment:
@huong-li-nguyen I don't think we should need - it suggests that your imports in `examples/kpi` are possibly not set up correctly. Or at least even if you do need to do some tomfoolery with Python paths, I don't think this is the right place to do it.

Reply:
I am just seeing this now 😬 This will become relevant again for the chart-gallery dashboard. Let's continue the discussion there.